src/pages/household/output/EarningsVariation/BaselineAndReformChart.jsx

```diff
@@ -184,10 +188,29 @@ function BaselineAndReformTogetherChart(props) {
       style={{
         width: "100%",
       }}
+      onHover={(data) => {
+        const netIncome = data.points[0].y.toLocaleString("en-US", 
+          { style:"currency",
+            currency:"USD",
+            maximumFractionDigits: 0
+          });
+        const employmentIncome = data.points[0].x.toLocaleString("en-US", 
+          { style:"currency",
+            currency:"USD",
+            maximumFractionDigits: 0
+          });
+        const message = `Net Income ${netIncome} Employment Income ${employmentIncome}`
```

Comment:
I tried adding a new line character `\n` to make Net Income and Employment Income appear on different lines on the Hovercard, but that had no effect on the displayed text. I also double checked the string that's passed to the Hovercard component if a `\n` is included  and confirmed with a `console.log` that the new line character is passed to the component correctly, despite the label text not displaying it.

Reply:
Yeah I think we might need to actually use a <br> tag for this rather than just a newline character?

Reply:
Thanks for the suggestion! Utilizing a `<br>` tag worked in splitting the string into the two lines.